test/GithubApi.Redirect.test.js

```diff
@@ -0,0 +1,27 @@
+const axios = require('axios');
+const { expect } = require('chai');
+const { StatusCodes } = require('http-status-codes');
+
+const repository = {
+  url: 'https://github.com/aperdomob/redirect-test'
+};
```

Comment:
It is not needed to define it as an object, you can define just the variable as a constant like:
```suggestion
const repositoryUrl ='https://github.com/aperdomob/redirect-test';
```
and use it with that value :)